bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -1520,6 +1563,45 @@ private CaseDbSchemaVersionNumber updateFromSchema7dot2toSchema8dot0(CaseDbSchem
 		}
 	}
 
+	/**
+	 * Updates a schema version 8.0 database to a schema version 8.1 database.
+	 *
+	 * @param schemaVersion The current schema version of the database.
+	 * @param connection    A connection to the case database.
+	 *
+	 * @return The new database schema version.
+	 *
+	 * @throws SQLException     If there is an error completing a database
+	 *                          operation.
+	 * @throws TskCoreException If there is an error completing a database
+	 *                          operation via another SleuthkitCase method.
+	 */
+	private CaseDbSchemaVersionNumber updateFromSchema8dot1toSchema8dot2(CaseDbSchemaVersionNumber schemaVersion, CaseDbConnection connection) throws SQLException, TskCoreException {
+		if (schemaVersion.getMajor() != 8) {
+			return schemaVersion;
+		}
+
+		if (schemaVersion.getMinor() != 1) {
+			return schemaVersion;
+		}
+		Statement statement = connection.createStatement();
+		acquireSingleUserCaseWriteLock();
+		try {
+			// create examiners table
+			if (this.dbType.equals(DbType.SQLITE)) {
+				statement.execute("CREATE TABLE tsk_examiners (examiner_id INTEGER PRIMARY KEY, login_name TEXT NOT NULL, display_name TEXT, UNIQUE(login_name) )");
+			} 
```

Comment:
is there ever a chance of different examiners having the same login name (say local accounts on different machines)?

Reply:
It's certainly possible to have same examiner login ids if they are local accounts and not in domain. 
Hopefully, people sharing cases with each other are not using local accounts though.

Otherwise we may need to get more sophisticated and determine the type of account, domain name or machine name, and prepend that. 

@bcarrier: should we open a JIRA to handle this later?

Reply:
I don't think we need a jira. We'll deal with it in the future if it comes up.